bin/fk.sh

```diff
@@ -100,6 +100,81 @@ frun()
   kit-run $repoDir/kit.yml $@
 }
 
+fup-help() {
+    echo "fk up [OPTIONS] repositoryName"
+    echo "Options: "
+    echo " --username|-u     github-username        Username undeer whom the repository will be created."
+    echo " --description|-d  \"description text\"   Repository description."
+    echo " --ssh                                    Configure remote origin to be over ssh."
+    echo " --private|-p                             Makes the repository private."
+    echo "If you select this option, Fantastic Kit is not repsonsible for any cost incurred by you or your organization."
+}
+
+fup() {
+    local username=$(fk config --key=gitUsername 2> /dev/null)
+    local private="false"
+    local description=""
+    local ssh=""
+    while [[ $1 ]]; do
+        case $1 in
+            --username|-u)
+                username=$2
+                shift
+                ;;
+            --private|-p)
+                private="true"
+                ;;
+            --ssh)
+                ssh="true"
+                ;;
+            --description|-d)
+                description=$2
+                shift
+                ;;
+            *)
+                break
+                ;;
+        esac
+
+        shift
+    done
+
+    local reponame=$1
```

Comment:
This assumes that the last thing that is not parsed is the reponame, didn't want to introduce a flag for this because it increases how much the user needs to type.

Reply:
Ideal scenario is fk up newRepo.